### PR TITLE
Adjust some contents in write_docs_en.rst for Contribue Documentation

### PR DESCRIPTION
**Pull request by weixing02**

### CODE REVIEW

File: doc/v2/dev/write_docs_en.rst [hunk 1]
```diff
@@ -2,21 +2,20 @@
...
+PaddlePaddle's documentation is built in two ways: using the PaddlePaddle.org tool and not using the PaddlePaddle.org tool. Both methods have their own advantages. The former facilitates previewing, while the latter facilitates debugging by the developer. And we could chose to build the documentation with docker or without docker in each way.
 
+We recommend using PaddlePaddle.org tool to build documentation.
 
-Use PaddlePaddle.org tool
---------------
-This is the recommended method to build documentation. It can compile documentation and preview the documentation in a web browser.
+With PaddlePaddle.org tool
+---------------------------
+This is the recommended method to build documentation, because it can automatically compile the documentation and preview the documentation directly in the web page, it should be noted that, in other ways, although you can preview the documentation, but the style of the documentation and the official website documentation is inconsistent, while Compiling with the PaddlePaddle.org tool produces a preview that is consistent with the official website documentation style.
```
Comment (luotao1) [inline doc/v2/dev/write_docs_en.rst]: Note that, in other ways, although you can preview the documentation, its style s inconsistent with the official website. While compiling with the PaddlePaddle.org tool produces a preview that is consistent with the official website documentation style.

Comment (weixing02) [reply doc/v2/dev/write_docs_en.rst]: Done

File: doc/v2/dev/write_docs_en.rst [hunk 2]
```diff
@@ -2,21 +2,20 @@
...
-When using the PaddlePaddle.org to compile documentations, the compiled documentations will be stored under a consolidated directory: .ppo_workspace/content
+PaddlePaddle's documentation includes both Chinese and English versions. The documentation is built using the ``cmake`` command to drive the ``sphinx`` compiler. The PaddlePaddle.org tool helps us to implement this compilation process and provides better preview results.
 
-How to Build Documentations
-============
+How to build Documentation
+===========================
 
-We recommend using PaddlePaddle.org tool to build documentation
+PaddlePaddle's documentation is built in two ways: using the PaddlePaddle.org tool and not using the PaddlePaddle.org tool. Both methods have their own advantages. The former facilitates previewing, while the latter facilitates debugging by the developer. And we could chose to build the documentation with docker or without docker in each way.
```
Comment (abhinavarora) [inline doc/v2/dev/write_docs_en.rst]: Lets make this sentence: **PaddlePaddle's documentation is built in two ways: using the PaddlePaddle.org tool and without using it.**

Comment (abhinavarora) [reply doc/v2/dev/write_docs_en.rst]: Drop the **And**  in the last sentence. The sentence will be:

We could **choose** to build the documentation with **Docker** or without it in each of the above ways.

File: doc/v2/dev/write_docs_en.rst [hunk 3]
```diff
@@ -2,21 +2,20 @@
...
 
+We recommend using PaddlePaddle.org tool to build documentation.
 
-Use PaddlePaddle.org tool
---------------
-This is the recommended method to build documentation. It can compile documentation and preview the documentation in a web browser.
+With PaddlePaddle.org tool
```
Comment (abhinavarora) [inline doc/v2/dev/write_docs_en.rst]: With -> **Using**

File: doc/v2/dev/write_docs_en.rst [hunk 4]
```diff
@@ -2,21 +2,20 @@
...
+---------------------------
+This is the recommended method to build documentation, because it can automatically compile the documentation and preview the documentation directly in the web page. Note that, in other ways, although you can preview the documentation, its style is inconsistent with the official website. While compiling with the PaddlePaddle.org tool produces a preview that is consistent with the official website documentation style.
```
Comment (abhinavarora) [inline doc/v2/dev/write_docs_en.rst]: directly in the web page -> directly in **a** web page

Comment (abhinavarora) [reply doc/v2/dev/write_docs_en.rst]: I think the last part of this paragraph should be as follows:

Note that,  although you can preview the documentation in other ways, its style may not be consistent with the official website. Compiling with the PaddlePaddle.org tool produces a preview that will be consistent with the official website documentation style.

File: doc/v2/dev/write_docs_en.rst [hunk 5]
```diff
@@ -2,21 +2,20 @@
...
 
-The tool uses Docker, please install it on your system. Please check Docker official website on how to install Docker. You may use the following commands to activate the tool
+The PaddlePaddle.org tool can be used with Docker and Docker needs to be installed first. Please refer to the `Docker's official website <https://docs.docker.com/>`_ on how to install Docker. After installing Docker, you may use the following command to activate the tool
```
Comment (abhinavarora) [inline doc/v2/dev/write_docs_en.rst]: Drop the **the** 
Please refer to **the** `Docker's -> Please refer to `Docker's

File: doc/v2/dev/write_docs_en.rst [hunk 6]
```diff
@@ -33,7 +32,7 @@ The tool uses Docker, please install it on your system. Please check Docker offi
     docker run -it -p 8000:8000 -v `pwd`:/var/content paddlepaddle/paddlepaddle.org:latest
 
 Note: PaddlePaddle.org will read the content repos specified in the -v (volume) flag of the docker run command
-Use a web browser and navigate to http://localhost:8000, click the buttons to compile the documentation
+Use a web browser and navigate to http://localhost:8000, click the buttons to compile the documentation.
```
Comment (abhinavarora) [inline doc/v2/dev/write_docs_en.rst]: navigate to http://localhost:8000, click the -> navigate to **http://localhost:8000.** **Click** the

File: doc/v2/dev/write_docs_en.rst [hunk 7]
```diff
@@ -58,19 +57,62 @@ If you don't wish to use Docker, you can also activate the tool through Django.
     pip install -r requirements.txt
     python manage.py runserver
 
+Specify the PaddlePaddle working directory for the environment variable CONTENT_DIR so that the tool could find where the working directory is.
+
 Use a web browser and navigate to http://localhost:8000, click the buttons to compile the documentation
 The compiled documentations will be stored in <paddlepaddle working directory>/.ppo_workspace/content
 
-If you want to learn more on the PaddlePaddle.org, please `click here <https://github.com/PaddlePaddle/PaddlePaddle.org/blob/develop/README.md>`_ 。
+Please `click here <https://github.com/PaddlePaddle/PaddlePaddle.org/blob/develop/README.md>`_ to get more informations about PaddlePaddle.org tool。
```
Comment (abhinavarora) [inline doc/v2/dev/write_docs_en.rst]: to get more informations about PaddlePaddle.org tool -> **for** more **information** about the PaddlePaddle.org tool

File: doc/v2/dev/write_docs_en.rst [hunk 8]
```diff
@@ -58,19 +57,62 @@ If you don't wish to use Docker, you can also activate the tool through Django.
...
+
+
+Without PaddlePaddle.org tool
```
Comment (abhinavarora) [inline doc/v2/dev/write_docs_en.rst]: The title should be

Manually Building the Documentation

File: doc/v2/dev/write_docs_en.rst [hunk 9]
```diff
@@ -58,19 +57,62 @@ If you don't wish to use Docker, you can also activate the tool through Django.
...
+-------------------------------
+
+Build PaddlePaddle's documentation with Docker，you need to install Docker first. Please refer to the `Docker's official website <https://docs.docker.com/>`_ on how to install Docker. After Docker is installed, you could use the scripts in the source directory to build the documentation.
+
+[TBD]
+
+If you do not wish to use Docker, you can also use the following command to directly build the PaddlePaddle documentation.
```
Comment (abhinavarora) [inline doc/v2/dev/write_docs_en.rst]: command -> **commands**

**Comment by abhinavarora** [comment]
Thank you for the PR. I have made some suggestions on the grammar and sentence structure for some sentences. Please let me know if you have any questions.

File: doc/v2/dev/write_docs_en.rst (hunk 2 above)
Comment (weixing02) [reply doc/v2/dev/write_docs_en.rst]: Done

File: doc/v2/dev/write_docs_en.rst (hunk 3 above)
Comment (weixing02) [reply doc/v2/dev/write_docs_en.rst]: Done

File: doc/v2/dev/write_docs_en.rst (hunk 4 above)
Comment (weixing02) [reply doc/v2/dev/write_docs_en.rst]: Done

File: doc/v2/dev/write_docs_en.rst (hunk 5 above)
Comment (weixing02) [reply doc/v2/dev/write_docs_en.rst]: Done

File: doc/v2/dev/write_docs_en.rst (hunk 6 above)
Comment (weixing02) [reply doc/v2/dev/write_docs_en.rst]: Done

File: doc/v2/dev/write_docs_en.rst (hunk 7 above)
Comment (weixing02) [reply doc/v2/dev/write_docs_en.rst]: Done

File: doc/v2/dev/write_docs_en.rst (hunk 8 above)
Comment (weixing02) [reply doc/v2/dev/write_docs_en.rst]: Done

File: doc/v2/dev/write_docs_en.rst (hunk 9 above)
Comment (weixing02) [reply doc/v2/dev/write_docs_en.rst]: Done

File: doc/v2/dev/write_docs_en.rst [hunk 10]
```diff
@@ -58,19 +57,62 @@ If you don't wish to use Docker, you can also activate the tool through Django.
...
+
+$processors indicates that as many processes as the CPU cores are started to compile in parallel. It should be set according to the number of CPU cores of your machine.
+
+After the compilation is complete, enter the ``doc/v2`` directory. If you chose to build documents, it will generate ``cn/html/`` and ``en/html`` subdirectories under this directory. If you chose to build APIs，it will generate``api/en/html`` subdirectory. Please enter these directories respectively and execute the following commands:
+
+.. code-block:: bash
+
+   python -m SimpleHTTPServer 8088
+
+Use a web browser and navigate to http://localhost:8000, you could see the compiled Chinese/English documents page and the English APIs page. The following figure is an example of the built English documents home page. Note that due to the sphinx's original theme used in the example, the style of the page is not consistent with the official website, but this does not affect the developer's debugging.
```
Comment (luotao1) [inline doc/v2/dev/write_docs_en.rst]: http://localhost:8088

File: doc/v2/dev/write_docs_cn.rst [hunk 11]
```diff
@@ -2,13 +2,14 @@
...
 
-PaddlePaddle的文档包括中英文两个部分。文档都是通过 ``cmake`` 驱动 ``sphinx`` 编译生成，也可以利用paddlepaddle.org工具来编译和预览文档。
+PaddlePaddle的文档包括中英文两个部分。文档都是通过 ``cmake`` 驱动 ``sphinx`` 编译生成的，PaddlePaddle.org工具可以帮助我们实现这一编译过程，并提供更好的预览效果。
 
 如何构建文档
 ============
 
 PaddlePaddle的文档构建有两种方式，分别为使用paddlepaddle.org工具和不使用paddlepaddle.org工具，两种方式都有各自的优点，前者方便预览，后者方便开发者进行调试。这两种方式中又分别有使用docker和不使用docker的两种构建方法。
 
+我们建议使用PaddlePaddle.org工具来构建文档。
```
Comment (luotao1) [inline doc/v2/dev/write_docs_cn.rst]: 35行：之后再用网页连到http://localhost:8000 （后面加个空格，这样会显示成网址效果）
同60行。

File: doc/v2/dev/write_docs_en.rst (hunk 10 above)
Comment (weixing02) [reply doc/v2/dev/write_docs_en.rst]: Done

File: doc/v2/dev/write_docs_cn.rst (hunk 11 above)
Comment (weixing02) [reply doc/v2/dev/write_docs_cn.rst]: Done

**Pull request merged**